app/models/service_template_ansible_playbook.rb

```diff
@@ -70,8 +73,8 @@ def create_new_dialog(dialog_name, job_template, hosts)
   private :create_new_dialog
 
   def self.create_job_templates(service_name, description, config_info, auth_user)
-    [:provision, :retirement, :reconfigure].each_with_object({}) do |action, hash|
-      next unless config_info[action] && config_info[action].key?(:playbook_id)
+    with_applied_config_info(CONFIG_ACTIONS, config_info, {}) do |action, hash|
```

Comment:
Does it really work? `each_with_object` will return the object. What does your block return?